api/handlers_api.go

```diff
@@ -62,6 +65,14 @@ func (r *Request) HandleCreateGame() mc.Message[mc.RespCreateGame] {
 	hostPlayer := game.CreateHostPlayer(r.Session.ID)
 	r.Session.Player = hostPlayer
 
+	ctx, cancel := context.WithTimeout(context.Background(), dbCtxTimeoutPeriod)
+	defer cancel()
+	q := sqlc.New(r.Session.Db)
```

Comment:
I don't know `sqlc` pretty well, but creating a new connection like this could be inefficient, does go have a generic connection pool?

Reply:
This is NOT a new connection.
it's a new struct of Queries that lets you do the operation.

That being said, that struct even can be created once